Health/Presentation/Onboarding/OnboardingViews/OnboardingVC/OnboardingCells/DiseaseCollectionViewCell.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  DiseaseCollectionViewCell.swift
+//  Health
+//
+//  Created by 권도현 on 8/8/25.
+//
+
+
+import UIKit
+
+class DiseaseCollectionViewCell: CoreCollectionViewCell {
+    
+    @IBOutlet weak var diseaseLabel: UILabel!
+    
+    override func setupHierarchy() {
+        super.setupHierarchy()
+        contentView.layer.cornerRadius = 12
```

Comment:
저희 확장 사용하시면 될 것 같아요!
```suggestion
        contentView.applyCornerStyle(.medium)
```